README.md

```diff
@@ -54,13 +54,13 @@ You can install from source:
 The [releases](https://github.com/fortio/fortio/releases) page has binaries for many OS/architecture combinations (see assets):
 
 ```shell
-curl -L https://github.com/fortio/fortio/releases/download/v1.39.1/fortio-linux_amd64-1.39.1.tgz \
+curl -L https://github.com/fortio/fortio/releases/download/v1.39.2/fortio-linux_amd64-1.39.2.tgz \
```

Comment:
will change to 1.40.0 once #688 merges and I rebase this to make the release/update flags